ui/00_message/skill/ability_info.xml

```diff
@@ -2264,6 +2332,9 @@ LV5～6: Increases stagger power.&lt;/COL&gt;</translation>
     <original>スラストのダメージ量が増加する
 &lt;COL ffdc78&gt;LV UP毎にダメージがより増加
 LV5～6で吹き飛ばし力が増加&lt;/COL&gt;</original>
+    <translation>Increases damage of Thrust.
+&lt;COL ffdc78&gt;LV UP: Increases damage further.
+LV5～6: Increases blow power.&lt;/COL&gt;</translation>
```

Comment:
Please change the fullwidth character to the regular character (in this case, ～ should be ~). This applies to any other translated messages as well.